src/main/java/org/snomed/snowstorm/core/data/services/CodeSystemService.java

```diff
@@ -484,6 +495,14 @@ public CodeSystemVersion findLatestVisibleVersion(String shortName) {
 		return null;
 	}
 
+	public CodeSystemVersion findPreviousVisibleVersion(String shortName) {
```

Comment:
The List returned from `findAllVersions(...)` is ordered and so I felt confident returning the first index. This is a similar approach to the `findLatestVisibleVersion` method.